services/users.service.js

```diff
@@ -148,7 +175,7 @@ module.exports = {
 				try {
 					const user = await this.adapter.findOne({ "apiKey.key": token });
 
-					return this.formatUser(user);
+					return user;
```

Comment:
Ezt itt nem értem, miért írtád át?

Reply:
A formatUser nekünk a response-ok miatt volt fontos, de ha már itt formatUser(user)-t térítek vissza, pl az _id mező is ki van szűrve belőle.